src/controllers/user.ts

```diff
@@ -0,0 +1,91 @@
+import { Request, Response, NextFunction } from 'express';
+import uuid from 'uuid/v1';
+import sequelize from '../data-access/sequelize';
+
+export const updateUserController = (req: Request, res: Response) => {
+    const { id, login, password, age } = req.body;
+
+    sequelize.models.user.findOne({
+        where: { id }
+    })
+    .then((data: any) => {
+        if (data) {
+            return sequelize.models.user.update(
+                {
+                    login,
+                    password,
+                    age,
+                    isDeleted: false,
+                },
+                {where: { id }}
+            )
+                .then(() => res.send('User has been updated'))
+                .catch(err => {
+                    console.log(err);
+            
+                    res.send('Database error');
+                });
+        }
+
+        res.send('User has not been found')
```

Comment:
REST service usually makes use of http statuses, 404, 500 - etc

Reply:
Fixed, thank you